config/lava/chromeos/chromeos-tast-template.jinja2

```diff
@@ -0,0 +1,49 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% block main %}
+{% set extra_kernel_args = "console_msg_format=syslog earlycon cros_debug cros_secure root=/dev/mmcblk0p3" %}
+{% do context.update({"extra_kernel_args": extra_kernel_args}) %}
```

Comment:
Oh I see, the `kernel-ci-base.jinja2` template doesn't do `context.update()`. So we could just have this here:
```suggestion
{% do context.update({"extra_kernel_args": "console_msg_format=syslog earlycon cros_debug cros_secure root=/dev/mmcblk0p3" }) %}
```
But that's fine, as eventually we should be using a `extra_kernel_args` variable. So we can merge it as-is I think.

Reply:
yes, the `kernel-ci-base.jinja2` template doesn't have context update :/ but I've updated considering your suggestion. I will open a PR to do parameters a bit more flexible as you suggested above. :)

Reply:
Have you submitted a job with the latest version of this PR? If so, please provide a LAVA job link to keep a trace.

Reply:
And it failed, but where is this kernel image coming from?
```yaml
      kernel:
        url: https://people.collabora.com/~alex.pereira/chromeos/chromeos-images/chromeos-20220209-4.14.243
```
There's a good known 4.14 Chrome OS kernel for octopus so we should be using that while setting things up in LAVA.

Reply:
Resubmitted it with the working kernel URL and that worked:
https://lava.collabora.co.uk/scheduler/job/5817798
(except the issue of /usr/local not being mounted but that's not an issue with this PR).

So I think this is ready to be merged.  I'd just like to get to the bottom of the kernel being used in the first job first.